aws/resource_aws_emr_cluster_test.go

```diff
@@ -1639,6 +1659,353 @@ resource "aws_iam_role_policy_attachment" "emr-autoscaling-role" {
 `, r, r, r, r, r, r, r, r, r, r)
 }
 
+func testAccAWSEmrClusterConfigInstanceFleets(r int) string {
+	return fmt.Sprintf(`
+provider "aws" {
+  region = "us-west-2"
+}
+
+resource "aws_emr_cluster" "tf-test-cluster" {
+  name          = "emr-test-%d"
+  release_label = "emr-5.8.0"
+  applications  = ["Spark"]
+
+  ec2_attributes {
+    subnet_id                         = "${aws_subnet.main.id}"
+    emr_managed_master_security_group = "${aws_security_group.allow_all.id}"
+    emr_managed_slave_security_group  = "${aws_security_group.allow_all.id}"
+    instance_profile                  = "${aws_iam_instance_profile.emr_profile.arn}"
+  }
+
+  instance_fleet = [
+  	{
+  	  instance_fleet_type = "MASTER"
+  	  instance_type_configs = [
+  	  	{
+  	  	  instance_type = "m3.xlarge"
+  	  	}
+  	  ]
+  	  target_on_demand_capacity = 1
+  	},
+  	{
+  	  instance_fleet_type = "CORE"
+  	  instance_type_configs = [
+  	  	{
+  	  	  bid_price_as_percentage_of_on_demand_price = 80
+  	  	  ebs_configuration {
+  	  	    ebs_block_device_configs = [
+  	  	      {
+  	  	        volume_specification {
```

Comment:
The EBS config definition used here is the old one. I'm surprised that this test still passes.